threedigrid_builder/grid/connection_nodes.py

```diff
@@ -24,14 +24,12 @@ class ConnectionNode:
     the_geom: shapely.Geometry
     code: str
     storage_area: float
-    manhole_id: int
+    is_manhole: bool
```

Comment:
Easy reference to whether a node is a manhole; `manhole_id` was used for this previously.

Reply:
Alvast iets dat ik maandag niet moet vergeten. Dit gaat invloed hebben op de live site 🤔

Reply:
Eenvoudige oplossing: `ConnectionNode.id` overnemen voor nodes met `is_manhole = True`.

Reply:
Opgelost in https://github.com/nens/threedigrid-builder/pull/390/commits/4e00db7e534a99132d674f4bdea4f77712c6d1e4 door manhole_id toe te voegen bij het schrijven naar gridadmin. Eerder toevoegen voegt imho enkel complexiteit nodig die niet nodig is.